content/Software Support/IDE Settings/About-mdns-discovery.md

```diff
@@ -3,7 +3,7 @@ title: "About mdns-discovery"
 id: 4506515275548
 ---
 
-Arduino IDE includes an [mDNS discovery tool](https://github.com/arduino/mdns-discovery) which is used to find available network ports.
+Arduino IDE includes an [mDNS discovery tool](https://github.com/arduino/mdns-discovery) which is used to find available network ports. These ports are essential for enabling communication between the IDE and connected devices, particularly for tasks like over-the-air (OTA) uploads. For example, the Board Manager and Library Manager rely on network ports to detect and manage Arduino boards and libraries connected to the network, ensuring seamless communication and updates. By using mDNS discovery, the Arduino IDE can dynamically detect the presence of compatible devices, making the development process more efficient and streamlined.
```

Comment:
@per1234 Would you mind having a look at this paragraph? This article sees a fair amount of traffic, and we have received consistent feedback asking for more information about the application and why/if they should allow it.